docs/api/queries/get.mdx

```diff
@@ -143,6 +143,12 @@ cy.get('form').within(() => {
 })
 ```
 
+#### Find an anchor element by its label/text, remove some attribute and click it
+
```

Comment:
It would be good to an add explanation of why this is useful. Something like:

```suggestion

The following may be useful to change remove a `target` of `"_blank"` before clicking a link that would otherwise launch a new tab/window: 

```